src/components/ClassList.tsx

```diff
@@ -0,0 +1,136 @@
+                                        </Box>
+                                        <Box
+                                            as="span"
+                                            color="gray.600"
+                                            fontSize="sm"
+                                            textTransform="capitalize"
+                                        >
+                                            {toTitleCase(item.weekday)}{" "}
+                                            {convertToAmPm(
+                                                item.startTimeMinutes,
+                                            )}
+                                            -
+                                            {convertToAmPm(
+                                                item.startTimeMinutes +
+                                                    item.durationMinutes,
+                                            )}
+                                        </Box>
+                                        <Box
+                                            as="span"
+                                            color="gray.600"
+                                            fontSize="sm"
+                                        >
+                                            Teacher: {item.teacherName}
+                                        </Box>
+                                    </VStack>
+                                </GridItem>
+                                <GridItem>
+                                    <Flex
+                                        p="6"
+                                        direction="column"
+                                        justify="center"
+                                        height="100%"
+                                    >
+                                        <Spacer />
+                                        <Badge
+                                            borderRadius="full"
+                                            textTransform="capitalize"
+                                            fontWeight="medium"
+                                            letterSpacing="wide"
+                                            backgroundColor="#0C53A0"
+                                            textAlign="center"
+                                            color="white"
+                                            pb="1"
+                                            pt="1.5"
+                                            px="3"
+                                        >
+                                            {item.ageGroup}
+                                        </Badge>
+                                        <Spacer />
+                                        <Box
+                                            as="span"
+                                            color="gray.600"
+                                            fontSize="sm"
```

Comment:
For the participant availability text, could you set the `textAlign` prop to `right`

Reply:
😭  i saw that but didnt know how to do it bc then i cant use grid

Reply:
omg sorry I don't know anything about flex container spacer etc. but I'm glad you figured it out (??)